src/.env_template

```diff
@@ -1,4 +1,5 @@
 DISCORD_TOKEN="YOUR DISCORD TOKEN"
 CLIENT_ID="YOUR APPLICATION ID"
 GUILD_ID="YOUR_SERVER_ID"
-MONGO_URI="MONGO_URI"
\ No newline at end of file
+MONGO_URI="MONGO_URI"
+GENERAL_CHANNEL_ID="YOUR SERVER'S GENERAL CHANNEL ID"
```

Comment:
Please change to:

- WELCOME_CHANNEL_ID="SERVER'S WELCOME CHANNEL ID"
- BOT_CHANNEL_ID="SERVER'S BOT CHANNEL ID"

Reply:
Corrected.